src/drift-monitor.ts

```diff
@@ -505,11 +505,11 @@ export class DriftMonitor extends Construct {
       return Duration.minutes(5);
     } else if (duration.toMinutes() <= 15) {
       return Duration.minutes(15);
-    } else if (duration.toHours() <= 1) {
+    } else if (duration.toMinutes() <= 60) {
```

Comment:
Curious why this change was needed?

Reply:
`Duration.minutes(59).toHours()` throws `Error: '59 minutes' cannot be converted into a whole number of hours.`. Basically duration throws when the target duration isn't whole numbers.

That means that users may encounter still errors when providing `runEvery: Duration.minutes(61)`, though I'm not sure why users will provide such durations.

Reply:
I recall some option of Duration to perform integral rounding

Reply:
Can't believe I've missed that, thanks! [pr#35](https://github.com/cdklabs/cdk-drift-monitor/pull/35) fixes the problem.